src/main/kotlin/fi/metatavu/timebank/api/severa/SeveraBearerTokenContainer.kt

```diff
@@ -0,0 +1,71 @@
+package fi.metatavu.timebank.api.severa
+
+import com.google.gson.Gson
+import com.google.gson.JsonSyntaxException
+import fi.metatavu.timebank.api.severa.models.SeveraAccessToken
+import okhttp3.FormBody
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.eclipse.microprofile.config.inject.ConfigProperty
+import org.slf4j.Logger
+import javax.enterprise.context.ApplicationScoped
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+
+@RequestScoped
+class SeveraBearerTokenContainer {
```

Comment:
This should be `@ApplicationScoped` instead.

This should be named `SeveraAccessTokenContainer` imo.

Also add missing docs and pad the class definition with an empty line 

 ```kotlin
 class SeveraAccessTokenContainer {
 
     @ConfigProperty(...
 ```
 
 This project might have a good production grade reference on how to implement this https://github.com/Metatavu/metaform-api/blob/develop/src/main/kotlin/fi/metatavu/metaform/server/keycloak/KeycloakControllerToken.kt